packages/calypso-e2e/src/lib/pages/gutenberg-editor-page.ts

```diff
@@ -149,13 +148,18 @@ export class GutenbergEditorPage extends BaseContainer {
 	 * @param {boolean} visit Whether to then visit the page.
 	 * @returns {Promise<void} No return value.
 	 */
-	async publish( { visit = false }: { visit?: boolean } ): Promise< void > {
+	async publish( { visit = false }: { visit?: boolean } = {} ): Promise< string > {
 		await this.frame.click( `${ selectors.editPostHeader } >> text=Publish` );
 		await this.frame.click( `${ selectors.publishPanel } >> text=Publish` );
+		const viewPublishedArticleButton = await this.frame.waitForSelector(
+			`${ selectors.publishPanel } a:has-text("View")`
```

Comment:
Can we make this a variable so we're not having to use the same thing in line 173?

Reply:
Gotcha, will push up a revised commit.